package/boot/uboot-rockchip/src/arch/arm/dts/rk3568-xgp-u-boot.dtsi

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+
+#include "rk3568-u-boot.dtsi"
+
+/ {
+	chosen {
+		stdout-path = &uart2;
+		u-boot,spl-boot-order = "same-as-spl", &sdmmc0, &sdhci;
+	};
+};
+
+&sdmmc0 {
+	bus-width = <4>;
+	u-boot,spl-fifo-mode;
+};
+
+&uart2 {
+	u-boot,dm-spl;
+	clock-frequency = <24000000>;
+	status = "okay";
+};
```

Comment:
```suggestion
```

these props are already defined in rk3568-u-boot.dtsi, drop

the dts name should match the dt compatible (`[<vendor>,]<board>`), i.e. `rk3568-xiguapi-v3-u-boot.dtsi`